net/transmission/Makefile

```diff
@@ -94,7 +93,7 @@ endef
 define Package/transmission-web
   $(call Package/transmission/template)
   TITLE+= (webinterface)
-  DEPENDS:=transmission-daemon
+  DEPENDS:=transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl
```

Comment:
`DEPENDS:=PACKAGE_transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl` doesn't work, only -polarssl opens -web.
The same result for PROVIDES/DEPENDS, which looks like a fundamental problem of the configuration system.
The only two variants that work are `DEPENDS:=transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl` and a more symmetrical one `DEPENDS:=@(PACKAGE_transmission-daemon-openssl||PACKAGE_transmission-daemon-polarssl)`